bin/sort-pr-by-date.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+
+if ( 2 < count( $argv ) ) {
+	echo "Incorrect # of arguments.\nUsage: php sort-pr-by-date.php 7273,2737,7273\n";
+	exit;
+}
+
+$command = 'git log --pretty=format:"%ad,%s" --all --date=local --author-date-order --reverse --grep ":prs" | sort -u | grep -o "\(#[0-9]\+\)"';
```

Comment:
`sort -u` breaks this command for me, as git returns the day first `Thu Aug 19...`, so sort only sorts by the days per week. This would break any PRs merged across multiple weeks.

The `git` command you have above already returns the correct order, so I don't think `sort -u` is necessary. It defaults from oldest to newest for me (which I think is what you want right? ) to reverse the list you can tag on ` | tail -r`